src/Microsoft.DotNet.Interactive.Formatting.Tests/FormatterTests.cs

```diff
@@ -170,10 +171,10 @@ public void Check_default_formatting_of_strings_with_escapes()
             {
                 var value = "hola! \n \t \" \" ' ' the joy of escapes! and    white  space  ";
 
-                var mimeType = Formatter.GetPreferredMimeTypeFor(typeof(string));
-                var text = value.ToDisplayString(mimeType);
+                var mimeType = Formatter.GetPreferredMimeTypesFor(typeof(string));
+                var text = value.ToDisplayString(mimeType.FirstOrDefault());

```

Comment:
The fact that it's first isn't important. Maybe select it out by name to make the test more robust?

Reply:
right